### PR TITLE
2.5.1 update

### PR DESCRIPTION
**Pull request by Scraayp**
Fixes:
- [x] Message embeds not working | #121 
- [x] Ban error message sends even when succesful | #117 
- [x] Bot crashes when unable to delete messages

Added: 
- [x] Remove bot messages after a specefic time

### CODE REVIEW

**Comment by Scraayp** [comment]
Ready for review

File: docs/index.html [hunk 1]
```diff
@@ -13,9 +13,9 @@
     <link type="text/css" rel="stylesheet" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
     <link type="text/css" rel="stylesheet" href="styles/prettify-tomorrow.css">
     <link type="text/css" rel="stylesheet" href="styles/jsdoc-default.css">
-    <link rel="icon" 
-      type="image/png" 
-      href="img/antispam.png">
+	<link rel="icon"
```
Comment (Michael-J-Scofield) [inline docs/index.html:16]: can you make sure the indentation is consistent here? and in general clean up this file

File: index.js [hunk 2]
```diff
@@ -271,25 +303,30 @@ class AntiSpamClient extends EventEmitter {
...
+        if (this.options.errorMessages) {
+          message.channel.send(this.format(this.options.banErrorMessage, message)).catch((e) => {
+            if (this.options.verbose) {
+              console.error(`DAntiSpam (banUser#sendSuccessMessage): ${e.message}`)
+            }
+          }).then(msg => {
+			return this.clearBotMessages(msg)
+		})
+        }
+      })
```
Comment (Michael-J-Scofield) [inline index.js:329]: Indentation issue here too? Please autoformat so we're sure its consistent :)

**Review by Michael-J-Scofield — CHANGES_REQUESTED** [review]
looks good, I think we can merge once the formatting is consistent.

**Review by Michael-J-Scofield — APPROVED** [review]

**Pull request merged**